### DISCUSSION TITLE
When do you plan to make sodium and oculas or iris compatible?

### DISCUSSION THREAD

**Discussion by Thunderrock424242** [Sinytra/Connector#130] [Q&A] [answered]
.

**Comment by Lolothepro** [comment, marked as answer]
Connector is already compatible with Rubidium 0.7.0a.
Unfortunately, Oculus is not currently compatible with Rubidium 0.7.0a.
(and for real Iris and Sodium, they will never be compatible
Sodium is incompatible with getRenderLayers, and this cannot be changed by Connector, in addition to being incompatible with ELS.
)
https://github.com/Sinytra/Connector/issues/71#issuecomment-1686544359
https://github.com/Sinytra/Connector/issues/39#issuecomment-1677957750

**Reply by Thunderrock424242** [reply]
Rubbium 0.7.0a does not work for me

**Reply by Lolothepro** [reply]
wdym?

**Reply by Thunderrock424242** [reply]
Connecter won't work with the new rubbium version I thought you said they are compatible

**Reply by Lolothepro** [reply]
Rubidium 0.7.0a works with Connector (are you sure you didn't leave Oculus in your mods folder?)

**Reply by Thunderrock424242** [reply]
Oculas is in my mods folder but I have it turned off

**Reply by Lolothepro** [reply]
![2023-08-27_19 20 58](https://github.com/Sinytra/Connector/assets/68275908/9175ea0b-c3f4-4ee5-b1fe-56e05a848c92)
![2023-08-27_19 21 41](https://github.com/Sinytra/Connector/assets/68275908/313afdab-dd81-4d09-96a9-6ccc697acfdf)

**Reply by Lolothepro** [reply]
> Oculas is in my mods folder but I have it turned off

turned off?

**Reply by Thunderrock424242** [reply]
Curseforge allows you to disable mods to find errors

**Reply by Lolothepro** [reply]
Can you send your log file?

**Reply by coco875** [reply]
do you have also other mod ? because he can create the conflict

**Reply by Thunderrock424242** [reply]
I sent the logs

**Reply by Thunderrock424242** [reply]
For some reason it created a new thread

**Comment by Thunderrock424242** [comment]
> Can you send your log file?

[crash-2023-08-27_18.47.34-client.txt](https://github.com/Sinytra/Connector/files/12448981/crash-2023-08-27_18.47.34-client.txt)
[latest.log](https://github.com/Sinytra/Connector/files/12448982/latest.log)

**Reply by coco875** [reply]
the lastest version of rubidium have some conflict
https://github.com/Asek3/Rubidium/issues/590 with create in inventory
https://github.com/Asek3/Rubidium/issues/586 with create in world
https://github.com/Asek3/Rubidium/issues/580 with valkeriens skies
and by looking this line
`Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Redirector renderShipChunkBlockEntity(Lnet/minecraft/client/renderer/blockentity/BlockEntityRenderDispatcher;Lnet/minecraft/world/level/block/entity/BlockEntity;FLcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/client/renderer/MultiBufferSource;Lcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/client/renderer/RenderBuffers;Lit/unimi/dsi/fastutil/longs/Long2ObjectMap;Lnet/minecraft/client/Camera;F)V in valkyrienskies-common.mixins.json:mod_compat.sodium.MixinSodiumWorldRenderer failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap valkyrienskies-1201-common-refmap.json`
it's a conflict between redirector, valkyrienskies
valkyrienskies is a mod in beta like connector, so disable every mod and enable just connector and rubidium and send a crash report to have proof to have a conflict between connector and rubidium

**Reply by Thunderrock424242** [reply]
[latest.log](https://github.com/Sinytra/Connector/files/12455268/latest.log)
rubbium works but what is causing the crash now?

**Reply by Lolothepro** [reply]
timestacker
